packages/razzle-plugin-svelte/index.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const WebpackConfigHelpers = require('razzle-dev-utils/WebpackConfigHelpers');
+const Helpers = new WebpackConfigHelpers(process.cwd());
+
+module.exports = function modify(config) {
```

Comment:
See the new https://razzle-git-next.jared.vercel.app/docs/customization#writing-plugins